templates/deno/remix.config.js

```diff
@@ -1,12 +1,16 @@
 module.exports = {
   serverBuildTarget: "deno",
   server: "./server.ts",
-
-  /*
-  If live reload causes page to re-render without changes (live reload is too fast),
-  increase the dev server broadcast delay.
-
-  If live reload seems slow, try to decrease the dev server broadcast delay.
-  */
+  /**
+   * If live reload causes page to re-render without changes (live reload is too fast),
+   * increase the dev server broadcast delay.
+   *
+   * If live reload seems slow, try to decrease the dev server broadcast delay.
+   */
```

Comment:
nit: using jsdoc-style comment is overkill and runs into the same problem as multiple `//` in that editing the comment requires messing with comment indicators on each line. Multi-line comment should be fine (`/*` and `*/`)